omero/apps/index.html

```diff
@@ -162,3 +162,17 @@ <h4>OMERO-<span class="lowercase">metadata</span></h4>
                 </a>
             </div>
         </div>
+        <div class="row">
+            <div class="medium-1 medium-offset-2 columns">
+                <i class="icon-fa-blue fa fa-share-alt fa-2x"></i>
+            </div>
+            <div class="omero-key-feature-container medium-7 columns end">
+                <h4>OMERO-<span class="lowercase">cli-duplicate</span></h4>
+                <p>
+                    Create data duplicates.
```

Comment:
"Duplicate Objects in OMERO, without duplicating binary data." ? (from the repo)

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/466/commits/15b5e3a46fc85993add921d6531a8c3790fc2771